files/setup.sh

```diff
@@ -43,6 +43,10 @@ CUSTOM_VEBA_TLS_PRIVATE_KEY=$(/root/setup/getOvfProperty.py "guestinfo.custom_tl
 CUSTOM_VEBA_TLS_CA_CERT=$(/root/setup/getOvfProperty.py "guestinfo.custom_tls_ca_cert")
 DOCKER_NETWORK_CIDR=$(/root/setup/getOvfProperty.py "guestinfo.docker_network_cidr")
 POD_NETWORK_CIDR=$(/root/setup/getOvfProperty.py "guestinfo.pod_network_cidr")
+SYSLOG_SERVER_HOSTNAME=$(/root/setup/getOvfProperty.py "guestinfo.syslog_server_hostname")
```

Comment:
Could there be any parsing/validation issue with these variables, e.g. incorrect user input with slashes, dots, value types etc.?

Reply:
These are pre-defined params just like other OVF keys which are rendered in vSphere UI, so these would not be something users are manually inputting